ror/.stickler.yml

```diff
@@ -0,0 +1,20 @@
+# add the linters you want stickler to use for this project
+linters:
+  rubocop:
+    display_cop_names: true
+    # indicate where is the config file for stylelint
+    config: './rubocop.yml'
+
+# add the files here you want to be ignored by stylelint
+files:
+  ignore:
```

Comment:
please make sure that ignore contains the same files as exclude here https://github.com/microverseinc/linters-config/pull/29/files#diff-bb818e2d66f2dd3502ac21100eb9d755R2